spec/requests/api/v0.1/put_trainee_spec.rb

```diff
@@ -199,6 +199,7 @@
 
             expect(response).to have_http_status(:ok)
             expect(trainee.first_names).to eq("Alice")
+            expect(response.parsed_body[:data][:study_mode]).to eq("63")
```

Comment:
I can't see where this value is being updated in the PUT request.

Reply:
Updated 👍